scripts/epa/airdata/criteria_gases/README.md

```diff
@@ -0,0 +1,33 @@
+# Importing EPA AirData - Criteria Gases
```

Comment:
nit: maybe note that the script downloads the CSV from the source?

Reply:
done